src/maintainer/knowledge_base.rst

```diff
@@ -687,11 +687,11 @@ In order to qualify as a noarch python package, all of the following criteria mu
   - No skips except for python version. If the recipe is py3 only, remove skip
     statement and add version constraint on python in ``host`` and ``run``
     section.
-  - 2to3 is not used
+  - ``2to3`` is not used
   - Scripts argument in setup.py is not used
-  - If ``console_script`` entrypoints are in setup.py, they are listed in meta.yaml
+  - If ``console_script`` entry points are defined in ``setup.py`` or ``setup.cfg``, they are also listed in the ``build`` section of ``meta.yaml``
```

Comment:
I would keep it one word b/c that is how people will find it in the `setup.*` files.

Reply:
The world is actually `entry_points`, which I had used at a certain point while editing this, but then I have to also quote it as code right after another code-quoted word. But I can do that if you prefer.

Reply:
Indeed. I think it is better.